client/signup/p2-step-wrapper/style-p2v2.scss

```diff
@@ -0,0 +1,158 @@
+@import url( 'https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;900&display=swap' );
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+@import 'calypso/assets/stylesheets/_p2-extends.scss';
+
+.is-p2v2 {
+	.p2-step-wrapper {
+		box-sizing: border-box;
+		margin: 0 auto;
+		height: 100vh;
+		display: flex;
+		flex-direction: column;
+		padding: 24px 1.5em 0;
+		font-family: var( --p2-font-inter );
+		line-height: 1.8;
+
+		@include break-mobile {
+			padding: 48px 20px 0;
+			max-width: 440px;
+		}
+
+		@include break-medium {
+			padding: 64px 20px 0;
+		}
+
+		@include break-huge {
+			padding: 80px 20px 0;
+		}
+
+		input {
```

Comment:
Nit: could we name all the elements and target them by class names, please?

Reply:
These rules were actually redundant and have been removed.